apps/wakunode2/app.nim

```diff
@@ -85,6 +85,38 @@ func node*(app: App): WakuNode =
 func version*(app: App): string =
   app.version
 
+## Retrieve dynamic bootstrap nodes (DNS discovery)
+
+proc retrieveDynamicBootstrapNodes*(dnsDiscovery: bool,
```

Comment:
I think is better to keep this in the `node_factory.nim` so that the `libwaku` could potentially benefit from it when we add this feature. wdyt @jm-clius ?

Reply:
Sounds good to me :) Will wait a bit before merging so we confirm it and if so I'll update the PR 🤩